Tests/MarathonTests/MarathonTests.swift

```diff
@@ -644,7 +644,7 @@ class MarathonTests: XCTestCase {
 
 fileprivate extension MarathonTests {
     func createFolder() -> Folder {
-        let folderName = ".marathonTests"
+        let folderName = ".marathonTests-\(NSUUID().uuidString)"
```

Comment:
This is awesome! But I suggest encapsulating all folders created this way in a parent folder (that we can still call `.marathonTests`). Then at the end of the test suite we clear up that folder. Otherwise, the risk is that, if you abort the tests in the middle, that you will have lots of `.marathonTests-XXXX` folders lying around in your home folder 😅